PrimeroAdministrationandConfigurationGuide.md

```diff
@@ -546,6 +610,32 @@ Once a Lookup is created, you can add to a field type and select whether you can
 
 To edit the properties of an existing Lookup, find the name and click _Edit_ in the options in the right column. From the _Show_ page, click the EDIT button at the top of the screen.
 
+## Jordan Case Identity
+
+This is for developers working on a configuration files. The ID in the cases list rows is either in the format ```case_ID_code/short_ID``` or just ```short_ID``` if the case_ID_code isn't set. This is set as an auto-populated field in the ```SystemSettings```.
+
+# Alerts sand Notifications
+
+Alerts are a way of letting the case owner know when someone else has changed something in a case such as updating a field in a form, adding an incident, adding service provision, or requesting approval for a form. Alerts appear in the form of a message at the top of the associated form with a red "!" next to the form name in the form selection side bar.
+
+* In ```SystemSettings```
+
+  * toggle show_alerts to hide or show all the notifications. This includes the case sidebar form indicators and the messages in the form view for new subforms (incidents and service provision), approvals (BIA, case plan, case closure), and changed forms (currently only for service detail forms)
+
+  * ```approval_forms_to_alert``` is a map that is used to associate form ids with approval alert types
+
+  * ```changes_field_to_form``` is a map of field names to forms (either one field to one form or one field to an array of many forms is accepted structure). This is used to decide which fields will trigger an alert for a form when changed by someone who isn't the case owner.
+
+# Permissions
+
+## Create a CP Incident from a CP Case
+
+This is a new permission to allow creating an incident details from a case you don't own. Add ```Permission::INCIDENT_DETAILS_FROM_CASE``` to the role you want to give this permission to.
```

Comment:
Actually, looking at this again, there is a way to add this permission from the UI rather than in the bundle. Document this, including a screenshot of the appropriate permission checkbox in the role form.